src/Illuminate/Support/Str.php

```diff
@@ -13,18 +13,25 @@ class Str
     use Macroable;
 
     /**
-     * The cache of snake-cased words.
+     * The cache of camel-cased words.
      *
      * @var array
      */
-    protected static $snakeCache = [];
+    protected static $camelCache = [];
 
     /**
-     * The cache of camel-cased words.
+     * The cache of human-readable words.
      *
      * @var array
      */
-    protected static $camelCache = [];
+    protected static $humanCache = [];
+
+    /**
+     * The cache of snake-cased words.
+     *
+     * @var array
+     */
+    protected static $snakeCache = [];
```

Comment:
Can you maybe revert the order here and only add the new property? Because these changes make it a little hard to read.